### PR TITLE
Update api version p6

### PR DESCRIPTION
**Pull request by andreipope**

### CODE REVIEW

File: static/samples/k8s/vsphere/px-pks-vsphere-shared-specs.yaml [hunk 1]
```diff
@@ -430,7 +430,7 @@ spec:
       port: 8099
       targetPort: 8099
 ---
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
```
Comment (rodrigc) [inline static/samples/k8s/vsphere/px-pks-vsphere-shared-specs.yaml]: This should be `apps/v1`

Comment (andreipope) [reply static/samples/k8s/vsphere/px-pks-vsphere-shared-specs.yaml]: @rodrigc It's updated.

**Comment by rodrigc** [comment]
@andreipope in future when you file a GitHub PR, please remember to put the JIRA ticket ID somewhere in the commit message.  That will ensure that when you merge the PR, the JIRA ticket will automatically get updated with the status of the GitHub PR.

**Comment by andreipope** [comment]
@rodrigc Will do. Thanks!

**Pull request merged**